src/style.css

```diff
@@ -1,3 +1,94 @@
+@import url("https://fonts.googleapis.com/css2?family=Manrope:wght@400;500;700&family=Roboto:wght@400;500&display=swap");
+
+:root {
+  --active-color-link: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0 50px;
+}
+
+.header__brand {
+  width: 40px;
+  height: 40px;
+  line-height: 0;
+}
+
+.header__logo {
+  width: 100%;
+  height: 100%;
+}
+
+/* .nav {
+  padding: 10px 0;
```

Comment:
<img width="1234" alt="Screenshot 2023-08-10 at 18 01 51" src="https://github.com/mate-academy/layout_moyo-header/assets/53228259/c570be17-5d95-4122-8fde-c063dd4c8e44">

did you redeploy?
I don't see changes, also delete comments